src/app/persons/experiences/experience-form/experience-form.component.ts

```diff
@@ -43,9 +43,19 @@ export class ExperienceFormComponent implements OnInit, OnChanges {
     this.rebuildFormGroup(changes.person.currentValue.experiences);
   }
 
+  getTitle(experience: AbstractControl<any,any>): string {
```

Comment:
Look at the https://medium.com/ngconf/using-typescript-getters-in-your-angular-app-da040ec7566d.

Reply:
method deleted, resolved with custom pipe